packages/components-react/src/Button.tsx

```diff
@@ -0,0 +1,21 @@
+import { ForwardedRef, forwardRef, PropsWithChildren } from 'react';
+import Sparkle from 'react-sparkle';
+
+export const Button = forwardRef(
+  ({ children, className, ...restProps }: PropsWithChildren<any>, ref: ForwardedRef<HTMLButtonElement>) => {
+    return (
+      <button ref={ref} className={className} type={'button'} {...restProps} style={{ position: 'relative' }}>
```

Comment:
The [same comment from that other PR](https://github.com/nl-design-system/rijkshuisstijl-community/pull/133#discussion_r1279889746) applies here: it is quite odd to have a CSS and Web component implementation that looks like a blue button, and then not use that class name for React and have a completely unstyled button in the React button story.

Reply:
Added the class name.